branchff

```diff
@@ -116,6 +116,16 @@ logrun mkdir -p $WORKDIR
 gitlib::sync_repo $K8S_GITHUB_SSH $TREE_ROOT || common::exit 1 "Exiting..."
 logrun cd $TREE_ROOT
 
+# Is the master branch in a reasonable state to fast forward over the branch?
+GIT_DESCRIBE=$(git describe)
+if [[ ! $GIT_DESCRIBE =~ ^v${RELEASE_BRANCH/release-/} ]]; then
```

Comment:
we tag the master branch with `vX.(Y+1).alpha.0` when we create the release branch, so I don't think this will work unless we stop doing that.

we want to know if there's a tag on master that's newer than the most recent tag on the release branch. I'm not sure how to get git to tell us that.